lib/timex.ex

```diff
@@ -770,27 +770,38 @@ defmodule Timex do
   Returns a boolean indicating whether the first `Timex.Comparable` occurs between the second
   and third.
 
-  By default, the `start`and `ending` bounds are *exclusive*. You can opt for inclusive bounds
-  by setting the `:inclusive` option to `true`.
-
+  By default, the `start` and `end` bounds are *exclusive*. You can opt for inclusive bounds with the
+  `inclusive: true` option.
+  To set just one of the bounds as inclusive, use the
+  `inclusive: :start` or `inclusive: :end` option.
   """
   @type between_options :: [
     inclusive: boolean
+    | :start
+    | :end
   ]
   @spec between?(Time, Time, Time, between_options) :: boolean | {:error, term}
   @spec between?(Comparable.comparable, Comparable.comparable, Comparable.comparable, between_options) ::
     boolean | {:error, term}
   def between?(a, start, ending, options \\ []) do
-    inclusive = Keyword.get(options, :inclusive, false)
+    [inclusive_start, inclusive_end] =
```

Comment:
@ckhrysze Can we make this a tuple rather than a list? I missed the review on this, sorry. As a rule I prefer tuples for a known-size set of things, with small N. It better reflects the purpose, access is constant, and is a smaller in-memory structure (though obviously for cases like this, that isn't really important). You can commit the change directly to master, no need to go through another PR.